src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,230 @@
+---
+title: SplitChunksPlugin
+contributors:
+  - sokra
+  - jeremenichelli
+related:
+  - title: "webpack 4: Code Splitting, chunk graph and the splitChunks optimization"
+    url: https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
+---
+
+Originally in webpack, chunks (and modules imported inside them) were connected by a parent-child relationship in the internal webpack graph.
```

Comment:
I would rephrase as (merged with the next lines as well):

> By default, chunks (and the modules inside them) are connected by direct parent-child relationships in the dependency graph. However, there are instances where a dependency may be duplicated across chunks. Previously, the `CommonsChunkPlugin` was use to dedupe these dependencies. In v4, the `optimization.splitChunks` and `optimization.runtimeChunk` configuration options provide a more intuitive interface.
>
> Here is how the new flow works...

Reply:
I think that stating that they are (present tense) linked by parent-child relationship is confusing/wrong since this has completely changed on the core of the package.